drafts/use-cases/2.api-documentation.md

```diff
@@ -0,0 +1,111 @@
+## API Documentation
+
+### Story
+As an application user
+I want my application to be able to perform some precomputations 
+So it can pre-generate some views and other UI elements to speed things up.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var apiDocumentation = client.getApiDocumentation("/api");
+for (var $class of apiDocumentation.supportedClasses) {
+    // generate a partial view for the _$class_
+}
+```
+
+### Details
+The documentation obtain may be a large payload,
+thus client is expected to store it temporarely at least for the current session.
+
+```
+GET /api?documentation
+
+HTTP 200 OK
+```
+```javascript
+{
+    "@id": "/api?documentation",
```

Comment:
Should be `/apidoc` or whatever was used in the pseudo-code above. Same for the `GET`